website-next/pages/_app.js

```diff
@@ -0,0 +1,7 @@
+import '../styles/globals.css'
+
+function MyApp({ Component, pageProps }) {
```

Comment:
this _app.js is the Next.js entry point. this is where we can intercept what Next is doing and override things if necessary.

This file is vanilla right now, it was generated like this.